docs/spec-and-docs/NewGlossary.md

```diff
@@ -0,0 +1,368 @@
+
+##### <u>eRA Commons</u>
+the [`electronic Research Administration`](https://public.era.nih.gov/commons) is an online interface where signing officials, principal investigators, trainees, and post-docs at institutions/organizations can access and share administrative information relating to research grants. It is the `designated ID provider` for the whitelist of DCCs.
+
+##### [FAIR](https://www.go-fair.org/fair-principles/)
+`Findable, Accessible, Interoperable, and Reusable`. Making data FAIR is the main goal of the DCCs.
+
+##### <u>FAIR Assessment</u>
+a process which evaluates the level of compliance of a dataset with the FAIR principles. The assessment may be performed with tools such as [FAIRshake](https://fairshake.cloud) or [FAIREvaluator](https://fairsharing.github.io/FAIR-Evaluator-FrontEnd/#!/).
+
+##### [FAIR Insignia]()
+a grid of colored squares developed to visually communicate FAIRness level. The [FAIR](#FAIR) insignia identifies areas of strength and weakness in the FAIRness level of digital objects, guiding digital object producers on how to improve the FAIRness of their products. FAIR insignia is an output of the [FAIRshake](#FAIRshake)
+
+<img src="https://github.com/nih-cfde/the-fair-cookbook/blob/master/content/images/logo/FAIRinsignia.png?raw=true" alt="drawing" style="border:1px solid black;" width="100"  align="top" />
+
+##### [FAIRshake](https://fairshake.cloud/)
+a tool produced for carrying out *FAIR Assessment*. Under the [CFDE](#CFDE), each data center’s inventory will be evaluated consistently based on FAIRshake, and the Coordinating Center will work with the individual Common Fund programs to adjust [FAIR](#FAIR) measures to meet the needs of the Common Fund. FAIRshake includes evaluating the FAIRness of digital objects including datasets, tools, and repositories.
+
+##### [Figshare](https://figshare.com/)
+an online open access repository where researchers can preserve and share their research outputs, including figures, datasets, images, and videos.
+
+##### <u>File</u>
+a `digital asset`, that is a type of digital object that each of the [DCCs](#DCC) hosts, such as genomic, metagenomic, RNA-Seq sequence data, physiological, and metabolic data or generic metadata electronic document.
+
+##### [Frictionless Data Package](http://frictionlessdata.io/data-package/)
+a format specification produced by the [Open Knowledge Foundation](https://okfn.org/) and supported by the Frictionlessdata.io organization. It aims to shorten the path from data to insight with a collection of specifications and software for the publication, transport, and consumption of data. This kills the cycle of find/improve/share that makes for a dynamic and productive data ecosystem.
+
+##### [GitHub](https://github.com/)
+an online hub for storing and sharing computer programs and other plain text files. The [CFDE](#CFDE) team uses it for storage, hosting websites, communications, and project management.
+
+##### [Globus Automate]()
```

Comment:
broken link

Reply:
I didn't test the links yet... this is purely an aesthetic/structural check-in to make sure this is moving in the right direction. At the bottom are what I believe are a bunch of things that need definitions and places in the list left by someone in the past. If the format and set-up looks okay, then the pruning can begin.

The link question: some of the entries in the glossary are hyperlinked themselves, some of the hyperlinks are in the definition instead... I think one convention should be followed but I would not love to 'choose' the wrong one and go change them all back. Right now it's a totally mixed bag.

Reply:
ahh. format is fine. 

I think the ones that are hyperlinks are links out to other websites? I don't have an opinion on that. whatever works.